### ISSUE TITLE
java.lang.NullPointerException: Rendering Block Entity

### ISSUE THREAD

**Issue by ghost** [CreativeMD/LittleTiles#705]
https://pastebin.com/KwhKVzS7

I cant login to server anymore it crashes instantly. 

I have no idea how that would have happened. But I see in the log something about littleframes and there I have only once taken down a frame and placed it again with undo nothing more.

**Comment by CreativeMD** [comment]
Duplicate of https://github.com/CreativeMD/LittleFrames/issues/15
